make.bat

```diff
@@ -59,8 +59,10 @@ for /f "usebackq tokens=*" %%i in (`"%VsWhereDir%\Microsoft Visual Studio\Instal
 	set InstallDir=%%i
 )
 
-if exist "%InstallDir%\Common7\Tools\vsdevcmd.bat" (
-	call "%InstallDir%\Common7\Tools\vsdevcmd.bat" -arch=%HostArch% -host_arch=%HostArch% -no_logo
+if exist %InstallDir%\Common7\Tools\vsdevcmd.bat (
```

Comment:
Previously it checked `if exist "..." (`, I think the quoting was needed because InstallDir could contain spaces, no?

Reply:
Yes, I made a mistake.